src/db_ledger.rs

```diff
@@ -278,9 +313,12 @@ impl LedgerColumnFamilyRaw for ErasureCf {
 pub struct DbLedger {
     // Underlying database is automatically closed in the Drop implementation of DB
     db: Arc<DB>,
-    meta_cf: MetaCf,
-    data_cf: DataCf,
-    erasure_cf: ErasureCf,
+    pub meta_cf: MetaCf,
```

Comment:
can you avoid making these pub?  we put some effort into making them non-pub so that rocksdb could be replaced

Reply:
oops, yup that's a bad merge